### DISCUSSION TITLE
Remember zoom level for all files

### DISCUSSION THREAD

**Discussion by Faguss** [WinMerge/winmerge#1432] [Ideas]
I find it inconvenient to zoom out every time I open a file. Could you save zoom value like in Notepad++?

**Comment by Gitoffthelawn** [comment]
See #1433

**Comment by sdottaka** [comment]
I've implemented this feature in [e477205](https://github.com/WinMerge/winmerge/commit/e4772056fe149931a3a7b0ef2dbe604f801975fc)

**Comment by Faguss** [comment]
Works in 2.16.23. Thank you
